aufgabe-012/src/main/java/Main.java

```diff
@@ -1,26 +1,43 @@
 public class Main {
 
-    public static void main(String[] args) {
-
-        final String daten = "Jenny:21,Max:18,Larissa:28,John:31";
-
-        // TODO: In dieser Aufgabe haben wir wieder ein Datenstring vorgeben. Es enthält Paaren:
-        //  * einen Namen
-        //  * und ein Alter
-        // Die Paare sind durch einen ":" getrennt, während die Daten selbst nochmal durch ein "," getrennt sind.
-        // Also sie entsprechen dem Format:
-        // Name:Alter,Name:Alter,Name:Alter,...
-        //
-        // Im ersten Schritt sollen die Datensätze voneinander getrennt werden. Es soll also mit dem "," gesplittet
-        // werden. Danach sollen die Namen ausgegeben werden ohne das Alter. Hierzu kann auch wieder split verwendet
-        // werden.
-        //
-        // Die Ausgabe soll mit System.out.println erfolgen und wie folgt aussehen:
-        // Namen:
-        // Jenny
-        // Max
-        // Larissa
-        // John
+  public static void main(String[] args) {
 
+    final String daten = "Jenny:21,Max:18,Larissa:28,John:31";
```

Comment:
Bitte Aufgabe 12 aus diesem PR entfernen und entsprechend in der root pom.xml deaktivieren.